src/SFA.DAS.Forecasting.Web/Orchestrators/ForcastingOrchestrator.cs

```diff
@@ -86,8 +84,36 @@ IAccountProjectionDataSession accountProjectionDataSession
             var d = _mapper.MapProjections(result);
 
             return d.Where(m => m.Date.IsAfterOrSameMonth(DateTime.Today) && (!_applicationConfiguration.LimitForecast || m.Date < BalanceMaxDate))
+                .OrderBy(m => m.Date)
                 .Take(48)
                 .ToList();
         }
+
+        private IDictionary<FinancialYear, ReadOnlyCollection<ProjectiontemViewModel>> CreateProjectionTable(List<ProjectiontemViewModel> accountProjection)
+        {
+            var dict = accountProjection.Select(m =>
+                {
+                    return Tuple.Create(new FinancialYear(m.Date), m);
+                }).GroupBy(m => m.Item1, new FinancialYearIEqualityComparer())
+                .ToDictionary(
+                    m => m.Key,
+                    v => v.Select(t => t.Item2).ToList().AsReadOnly());
+
+            foreach(var s in dict )
+            {
+                s.Key.FirstStartDate = s.Value.OrderBy(m => m.Date).First().Date;
+                s.Key.LastEndDate = s.Value.OrderBy(m => m.Date).Last().Date;
+            };
+
+            return dict;
+        }
+
+        private async Task<Tuple<decimal, decimal>> GetBalance(string hashedAccountId)
+        {
+            var accountId = _hashingService.DecodeValue(hashedAccountId);
+            var balance = await _balanceDataService.Get(accountId);
+
+            return Tuple.Create(balance.Amount, balance.UnallocatedCompletionPayments);
```

Comment:
Can this just return the balance object instead? Its a bit more readable when used in the balance method